x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/logical_condition/components/condition_entry.tsx

```diff
@@ -130,6 +130,7 @@ export const ConditionEntry = memo<ConditionEntryProps>(
         alignItems="center"
         direction="row"
         data-test-subj={dataTestSubj}
+        responsive={false}
```

Comment:
curious why this is set to false rather than true?

Reply:
@parkiino we want the flexGroup to not snap to mobile view when in small screens, so that it continues to maintain its layout regardless of screen size.